lib/fitgem/client.rb

```diff
@@ -229,8 +148,7 @@ def post(path, body='', headers={})
       end
 
       def raw_post(path, body='', headers={})
-        headers.merge!('User-Agent' => "fitgem gem v#{Fitgem::VERSION}", 'Accept-Language' => @api_unit_system,
-                       'Accept-Locale' => @api_locale)
+        headers.merge! default_headers
         uri = "/#{@api_version}#{path}"
         access_token.post(uri, body, headers)
```

Comment:
The signature of `#post` changed in the OAuth2 gem. It's now only 2 parameters: `def post(path, opts = {}, &block)` (https://github.com/intridea/oauth2/blob/master/lib/oauth2/access_token.rb#L120). The other methods likely have the same issue. All the request method parameters are the same in the OAuth2 gem (`(path, opts = {}, &block)`).